mayastor-test/test_replica.js

```diff
@@ -161,9 +160,9 @@ describe('replica', function () {
 
   it('should not create a pool with invalid block size', (done) => {
     client.createPool(
-      { name: POOL, disks: disks, block_size: 1238513 },
+      { name: POOL, disks: disks.map((d) => `${d}?blk_size=1238513`) },
       (err) => {
-        assert.equal(err.code, grpc.status.INVALID_ARGUMENT);
+        assert.equal(err.code, grpc.status.INTERNAL);
```

Comment:
Is there a chance that we could preserve invalid_argument error that is better fit than internal error for the case of invalid block size specification?

Reply:
Yes, but create_or_import was simply enforcing a 0/512/4096 value on the block_size field, and returning INVALID_ARGUMENT otherwise, but it was actually using blk_size encoded in the URL so enforcement was happening in the bdev module so any error from there becomes INTERNAL.